src/index.html

```diff
@@ -10,13 +10,106 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
```

Comment:
The `alt` attribute for the logo image should be descriptive. Instead of `Logo`, describe what the logo is about. For example, if it's a company logo, mention the company name.